src/Blockcore.Explorer/ClientApp/src/app/nav-menu/nav-menu.component.html

```diff
@@ -19,7 +19,10 @@
         [routerLink]="[setup.current]">Ticker</a>
     </li>
     <li class="nav-item" *ngIf="setup.featureEnabled(setup.Explorer?.Features?.Explorer)">
-      <a class="nav-link" [routerLinkActive]="['link-active']" [routerLink]="[setup.current, 'explorer']">Explorer</a>
+      <a class="nav-link" [routerLinkActive]="['link-active']" [routerLinkActiveOptions]="{exact:true}" [routerLink]="[setup.current, 'explorer']">Explorer</a>
+    </li>
+    <li class="nav-item" *ngIf="setup.Chain.Symbol == 'CRS'">
```

Comment:
Should also include "TCRS" here?

Reply:
Ah maybe i should add it as a feature then?

Reply:
Yes, please do that. Make a feature called "SmartContract" or something that makes sense.

Reply:
It will have to be in reverse though to how features work it's disabled unless there is a sc feature

Reply:
You can flip the check easily in the UI code =) "!"

Reply:
Done